onnxruntime/core/graph/graph.cc

```diff
@@ -2517,7 +2517,9 @@ Status Graph::VerifyNodeAndOpMatch(const ResolveOptions& options) {
         }
         ORT_CATCH(const std::exception& ex) {
           ORT_HANDLE_EXCEPTION([&]() {
-            status = ORT_MAKE_STATUS(ONNXRUNTIME, INVALID_GRAPH, "This is an invalid model. Error in Node:", node_name, " : ", ex.what());
+            std::ostringstream err_msg;
+            err_msg << "This is an invalid model. In Node, " << node << ", Error ";
```

Comment:
Sometimes, NodeArg's name can be `0x12345`, so we use the new `operator<<` to print more information in `Node`.

Reply:
I think ORT_MAKE_STATUS (i.e. MakeString()) also uses `operator<<`